pymc/distributions/distribution.py

```diff
@@ -122,6 +122,8 @@ def __new__(cls, name, bases, clsdict):
                     )
 
             class_change_dist_size = clsdict.get("change_dist_size")
+            if class_change_dist_size is not None:
+                raise ValueError("HAHAHA")
```

Comment:
Fun debug statement left :)
```suggestion
```

Reply:
Hehe! That's what I get for not adding my changes line by line for this particular commit!